src/sched/job_runner.h

```diff
@@ -30,6 +30,14 @@ class JobRunner : public std::enable_shared_from_this<JobRunner> {
         std::chrono::nanoseconds active_time_{0};
     };
 
+    struct TryRunImmediately {
+        enum class State {
+            FAILED,
```

Comment:
label it

Reply:
Not necessary. We don't care about the numeric values.

Reply:
We do, for:
- logging/debugging.
- stability
- allow 0 or not, which can impact default-init/uninit vars. Yes they should not exist at all, but our code is not bug free either.
- for ipc or ffi. This piece won't ever need any of that (other than logging), but I don't want to rely on our dev to make the correct decision every time.